UefiPayloadPkg/Library/FdtParserLib/FdtParserLib.c

```diff
@@ -272,6 +274,10 @@ ParseReservedMemory (
       } else if (AsciiStrnCmp (TempStr, "runtime-data", AsciiStrLen ("runtime-data")) == 0) {
         DEBUG ((DEBUG_INFO, "  runtime-data"));
         BuildMemoryAllocationHob (StartAddress, NumberOfBytes, EfiRuntimeServicesData);
+      } else if (AsciiStrnCmp (TempStr, "soft-reserve", AsciiStrLen ("soft-reserve")) == 0) {

```

Comment:
Have we concluded the "soft-reserve" is the keyword for this type? how about "specific-purpose" which is more aligning with what UEFI spec defined?
Do we want to add this in UPL spec?

From UEFI spec:
EFI_MEMORY_SP Specific-purpose memory (SPM). The memory is earmarked for
specific purposes such as for specific device drivers or applications.
The SPM attribute serves as a hint to the OS to avoid allocating this
memory for core OS data or code that can not be relocated.
Prolonged use of this memory for purposes other than the intended
purpose may result in suboptimal platform performance.

Reply:
Yes- we should add it to UPL spec. And I am fine with PI aligned name. Updated accordingly.

Reply:
spec update is in progress: https://github.com/UniversalPayload/spec/pull/16